packages/connect-examples/expo-example/src/views/FirmwareScreen/DeviceField.tsx

```
@@ -38,7 +42,7 @@ function DeviceFieldView({ field, value }: DeviceFieldProps) {
         {`${field}: `}
       </Text>
       <Stack flex={1} paddingStart={4}>
-        {fieldValue?.startsWith('http') ? (
+        {fieldValue && fieldValue.startsWith && fieldValue?.startsWith('http') ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)